src/util/log.h

```diff
@@ -3,18 +3,17 @@
 #include "../math/types.h"
 #include "color.h"
 
-enum class LogLevel {
-    TRACE,
-    INFO,
-    WARNING,
-    ERROR,
-    NOTHING,
-};
+namespace LogLevel {
+static const u32 TRACE   = 1 << 0;
+static const u32 INFO    = 1 << 1;
+static const u32 WARNING = 1 << 2;
+static const u32 ERROR   = 1 << 3;
+}
```

Comment:
It should still be an enum. "static const" is not the solution here. Might also be good to add "ALL" and "NONE". Just for clarity.

Reply:
Nope, this is the simplest solution. NVM me.

Reply:
`ALL` as 0 I can get behind.